app/controllers/braintree/payment_method_nonces_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Braintree::PaymentMethodNoncesController < ApplicationController
+
+  def create
+    render text: Braintree::PaymentMethodNonce.create(params[:payment_method_id]).payment_method_nonce.nonce
+  end
```

Comment:
Yea, in general we need some integration tests with this whole thing as we are just filling in callbacks in various places so unit tests are of questionable value. In the interest of speed to deployment I didn't embark on that sort of integration tests. I could create a controller test here but I would really have to mock out all the braintree stuff so not sure it would be of much value either.